### PR TITLE
Added context mocks for easier testing.

### PR DESCRIPTION
**Pull request by rafalmnich**

### CODE REVIEW

File: mocks/mocks.go [hunk 1]
```diff
@@ -1,7 +1,13 @@
 package mocks
 
 import (
+	"fmt"
+	"os"
+
 	"github.com/stretchr/testify/mock"
+	"gopkg.in/urfave/cli.v1"
+
```
Comment (michalkurzeja) [inline mocks/mocks.go]: Imports. This lib doesn't separate internal imports.

File: mocks/mocks.go [hunk 2]
```diff
@@ -25,3 +31,33 @@ func (m *Logger) Error(msg string, ctx ...interface{}) {
...
+	ctx, err := clix.NewContext(cCtx)
+	if err != nil {
+		panic(err)
+	}
+
+	return ctx
+}
+
+func InitCliContext(args map[string]string, flags []cli.Flag) *cli.Context {
+	cliArgs := os.Args[0:1]
```
Comment (michalkurzeja) [inline mocks/mocks.go]: Why is this needed?

Comment (rafalmnich) [reply mocks/mocks.go]: I don't know - pasted your code ;)

File: mocks/mocks_test.go [hunk 3]
```diff
@@ -0,0 +1,76 @@
+package mocks_test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"gopkg.in/urfave/cli.v1"
+
+	"github.com/msales/pkg/v3/clix"
```
Comment (michalkurzeja) [inline mocks/mocks_test.go]: Same re imports.
